### ISSUE TITLE
Document the requirement for TCP passthrough when setting up Load Balancer

### ISSUE THREAD

**Issue by gitamiller** [docker/docker.github.io#6224]
File: [datacenter/ucp/2.2/guides/admin/install/plan-installation.md](https://docs.docker.com/datacenter/ucp/2.2/guides/admin/install/plan-installation/), CC @JimGalasyn 

It appears UCP and DTR require TCP passthrough and won't work if the Load Balancer is doing Application Load Balancing (ALB / Layer 7). If this is the case, we should make sure it's documented.

**Comment by anweiss** [comment]
We need to re-open this. This doc is no longer present in the UCP 3.0 docs. Furthermore the DTR 2.5.0 load balancing doc at [ee/dtr/admin/configure/use-a-load-balancer.md](https://docs.docker.com/ee/dtr/admin/configure/use-a-load-balancer/) makes no mention of HTTPS termination (aka passthrough). That also needs to be addressed.

**Comment by joaofnfernandes** [comment]
[In the UCP docs](https://docs.docker.com/ee/ucp/admin/configure/join-nodes/use-a-load-balancer/), there's a line there that states

> Since Docker UCP uses mutual TLS, make sure you configure your load balancer to:
>  * ...
> * Not terminate HTTPS connections.

As for DTR, I've filed docker/docker.github.io#6662 to address your feedback.
@anweiss please consider filing PRs in the future so that the problem gets addressed immediately. Unfortunatelly we a really understaffed, so issues might pile up for a while until they get resolved.

**Comment by anweiss** [comment]
Thanks @joaofnfernandes, seems unintuitive to have to look under the "Join nodes to your cluster" section in order to find the new location of the load balancer docs. However, I get, conceptually, why it makes sense there. Appreciate you triaging.

Will do my best to submit PRs for issues like this as well.

**Comment by joaofnfernandes** [comment]
I agree that the place is not intuitive. As we continue to add more and more articles about how to configure UCP, the harder it is to find the information you need.

We've done a bunch of exercises to try to group information in a more meaningful way. Some information got easier to find,  while other got harder 😭
